src/v/kafka/server/handlers/create_topics.cc

```diff
@@ -193,7 +205,7 @@ ss::future<response_ptr> create_topics_handler::handle(
           "Broker not available - audit system failure",
           std::move(response),
           std::move(request));
-
+        // TODO(oren): add properties here?
```

Comment:
question: purpose of todo?  will this be a follow up at some point?

Reply:
No sorry, it's a literal question. Should we add these values to the response here?

Reply:
Can you check what kafka does? In this conditional redpanda is going to return a non retryable error. In my opinion I don't think it matters that we return the properties here, but looks like I added the topic configs there.

Reply:
discussed in slack - gonna add it on the basis that configs are already there.

Reply:
done